From 188c0c31711e7ca34f7f395c2216ad5b5a4f8349 Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Mon, 26 Mar 2012 14:06:42 +0800
Subject: ENGR00177884-2 mx6q sabresd: config USB pin according to board

Add ENET_RX_ER__ANATOP_USBOTG_ID iomux setting in head file.

Signed-off-by: make shi <b15407@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6q.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6q.h b/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
index 6febfeb..ec6b228 100644
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
@@ -1631,6 +1631,8 @@ typedef enum iomux_config {
 #define _MX6Q_PAD_ENET_REF_CLK__ANATOP_USBPHY1_TSTO_RX_SQUELCH		\
 		IOMUX_PAD(0x04E8, 0x01D4, 7, 0x0000, 0, 0)
 
+#define _MX6Q_PAD_ENET_RX_ER__ANATOP_USBOTG_ID 				\
+		IOMUX_PAD(0x04EC, 0x01D8, 0, 0x0000, 0, 0)
 #define _MX6Q_PAD_ENET_RX_ER__ENET_RX_ER				\
 		IOMUX_PAD(0x04EC, 0x01D8, 1, 0x0000, 0, 0)
 #define _MX6Q_PAD_ENET_RX_ER__ESAI1_HCKR				\
@@ -5235,6 +5237,8 @@ typedef enum iomux_config {
 #define  MX6Q_PAD_ENET_REF_CLK__ANATOP_USBPHY1_TSTO_RX_SQUELCH		\
 		(_MX6Q_PAD_ENET_REF_CLK__ANATOP_USBPHY1_TSTO_RX_SQUELCH | MUX_PAD_CTRL(NO_PAD_CTRL))
 
+#define  MX6Q_PAD_ENET_RX_ER__ANATOP_USBOTG_ID	\
+		(_MX6Q_PAD_ENET_RX_ER__ANATOP_USBOTG_ID | MUX_PAD_CTRL(NO_PAD_CTRL))
 #define  MX6Q_PAD_ENET_RX_ER__ENET_RX_ER		\
 		(_MX6Q_PAD_ENET_RX_ER__ENET_RX_ER | MUX_PAD_CTRL(NO_PAD_CTRL))
 #define  MX6Q_PAD_ENET_RX_ER__ESAI1_HCKR		\
-- 
1.7.9.5

